### ISSUE TITLE
COPY command on FileBuilder producing invalid output

### ISSUE THREAD

**Issue by joachim-heck** [mariotoffia/FluentDocker#187]
Hey Mario, first of all thanks for that nice library and all the effort you put in.

I think I discovered two bugs in the FileBuilder when using the COPY command...

```
   var fileBuilder = new Builder()
                    .DefineImage(_imageName)
                    .From("mcr.microsoft.com/dotnet/sdk")
                    .UseWorkDir("/worker")
                    .Copy("entrypoint.sh", "/worker/entrypoint.sh")
                    .Build();
```

1.) the minor one is located in `FileBuilder.cs`:
```
 private void RenderDockerfile(TemplateString workingFolder)
    {
     // if (Directory.Exists(workingFolder)) <== MISSING !
       if (!Directory.Exists(workingFolder))
      {
        Directory.CreateDirectory(workingFolder);
      }
...
```
where you probably want to check whether the directory exists - otherwise create it.

If I follow the path of creating the temp. Dockerfile in order to build the image the output of the `RenderDockerfile(TemplateString workingFolder)` content looks like
```
FROM mcr.microsoft.com/dotnet/sdk
COPY [entrypoint.sh,/worker/entrypoint.sh]
```

Which finally leads to an exception in the docker build process stating: 
`Error response from daemon: dockerfile parse error line 2: COPY requires at least two arguments, but only one was provided. Destination could not be determined.
`

I think the correct out put should look like:

```
FROM mcr.microsoft.com/dotnet/sdk
COPY  ["entrypoint.sh","/worker/entrypoint.sh"]
```

Or am I understanding anything wrong?
Greets, Jo

**Comment by mariotoffia** [comment]
@joachim-heck Thanks for the bug report! I'll fix it during this week! :)

**Comment by joachim-heck** [comment]
Great, thanks a lot!

**Comment by mariotoffia** [comment]
@joachim-heck I've released [2.10.5](https://www.nuget.org/packages/Ductus.FluentDocker/2.10.5) - it is on indexing so it may take some time before nuget will publish it. 

Have a go and see if it works for you now.

Cheers,
 Mario :)

**Comment by joachim-heck** [comment]
Great - thanks for the fast fix!
